pkg/cmd/dbshell.go

```diff
@@ -50,20 +53,28 @@ var dbShellCmd = &cobra.Command{
 	},
 	RunE: func(_ *cobra.Command, args []string) error {
 
-		fetchObject := &kubernetes.KubeObject{Top: &summonv1beta1.SummonPlatform{}}
-		namespace := kubernetes.ParseNamespace(args[0])
-		err := kubernetes.GetObject(kubeconfigFlag, args[0], namespace, fetchObject)
+		// Determine if we are trying to connect to a microservice or summonplatform db
+		instance := strings.ToLower(args[0])
+		var namespace string
+		if strings.HasPrefix(instance, "svc-") {
```

Comment:
Could move this logic into the ParseNamespace func, we may want to re-use it elsewhere later. :)

Reply:
True! Probably expand it to other commands like shell, pyshell, etc.